src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.NormalPriorityProcessor.cs

```diff
@@ -451,16 +451,16 @@ private async Task ProcessReanalyzeDocumentAsync(WorkItem workItem, Document doc
                         }
                     }
 
-                    private void RemoveDocument(DocumentId documentId)
+                    private Task RemoveDocumentAsync(DocumentId documentId, CancellationToken cancellationToken)
                     {
-                        RemoveDocument(Analyzers, documentId);
+                        return RemoveDocumentAsync(Analyzers, documentId, cancellationToken);
                     }
 
-                    private static void RemoveDocument(ImmutableArray<IIncrementalAnalyzer> analyzers, DocumentId documentId)
+                    private static async Task RemoveDocumentAsync(ImmutableArray<IIncrementalAnalyzer> analyzers, DocumentId documentId, CancellationToken cancellationToken)
```

Comment:
>private [](start = 20, length = 7)

Curious, why split it into two methods?

Reply:
I didn't change this.  This is how the work coordinator worked before.

Reply:
resolving as not part of this review (this was a merge in from another branch).